Tasks/Common/AzureRmDeploy-common/operations/KuduServiceUtility.ts

```diff
@@ -125,6 +125,7 @@ export class KuduServiceUtility {
             console.log(tl.loc('PackageDeploymentInitiated'));
 
             let queryParameters: Array<string> = [
+                'isAsync=true',
```

Comment:
how is this consumed?

Reply:
this will be passed like this in zipdeploy api https://github.com/projectkudu/kudu/wiki/REST-API#zip-deployment

Reply:
nit: maybe add a comment pointing to the doc. will help someone else looking at it a later point in time.

Reply:
added comment